.ci/test-windows.ps1

```diff
@@ -110,7 +110,8 @@ if ($env:TASK -eq "regular") {
     Get-ItemProperty -Path Registry::HKEY_LOCAL_MACHINE\SOFTWARE\Khronos\OpenCL\Vendors
 
     conda activate $env:CONDA_ENV
-    sh "build-python.sh" bdist_wheel --integrated-opencl ; Assert-Output $?
+    # TODO: restore --integrated-opencl as part of https://github.com/microsoft/LightGBM/issues/6968
+    sh "build-python.sh" bdist_wheel ; Assert-Output $?
```

Comment:
Proposing that we temporarily skip the integrated OpenCL stuff for now, to unblock CI, and try to work on #6968 to restore that before the next release.